flow/platforms/asap7/config.mk

```diff
@@ -63,9 +63,12 @@ export SET_RC_TCL              = $(PLATFORM_DIR)/setRC.tcl
 
 # Route options
 export MIN_ROUTING_LAYER       ?= M2
-#export MIN_CLOCK_ROUTING_LAYER = M4
+export MIN_CLK_ROUTING_LAYER   ?= M4
 export MAX_ROUTING_LAYER       ?= M7
 
+# Define fastRoute tcl
+export FASTROUTE_TCL ?= $(PLATFORM_DIR)/fastroute.tcl
```

Comment:
So now set_macro_extension is no longer invoked.

Is that intentional?

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/2a81f92b390a6c18a09d0c1bab63fae846072a92/flow/scripts/util.tcl#L16-L26


I take it this is somehow related to https://github.com/The-OpenROAD-Project/OpenROAD/issues/7371 ?

Reply:
The main motivation is that I believe the default derates are likely too high.  All other PDKs have a fastroute.tcl with lower values.  The macro extension is probably not intended though if we don't need it that's fine.  Its mostly a workaround for drt issues we have seen in some PDKs.

Reply:
@oharboe The MACRO_EXTENSION is not used by any design, so it shouldn't havev any effect.

Regarding https://github.com/The-OpenROAD-Project/OpenROAD/issues/7371, I intend to review what we're calling regarding GRT configuration and do any necessary cleanup before storing them in ODB. I should start creating PRs for it next week.